tsconfig.json

```diff
@@ -9,5 +9,9 @@
   },
   "include": [
     "./**/*.ts"
+  ],
+  "exclude": [
+    "node_modules",
+    "dist"
```

Comment:
Why is this needed?

Reply:
Otherwise TypeScript will try to build `dist` again, but that results in errors since it's also writing to `dist`.

Reply:
It's not really needed now that I've added `rimraf` I guess, but it doesn't hurt to add it.